packages/design-system/src/components/InlineMessage/InlineMessage.stories.tsx

```diff
@@ -1,148 +1,105 @@
+	<StackVertical gap={'XS'}>
+		<InlineMessageInformation
+			title="Lorem ipsum"
+			withBackground
+			link={{ href: 'https://talend.com', children: 'Learn more' }}
+			description="dolor sit amet, consectetur adipiscing elit. Integer gravida orci lacus, vel convallis enim tposuere ac."
+		/>
+		<InlineMessageSuccess
+			title="Lorem ipsum"
+			withBackground
+			link={{ href: 'https://talend.com', children: 'Learn more' }}
+			description="dolor sit amet, consectetur adipiscing elit. Integer gravida orci lacus, vel convallis enim tposuere ac."
+		/>
+		<InlineMessageWarning
+			title="Lorem ipsum"
+			withBackground
+			link={{ href: 'https://talend.com', children: 'Learn more' }}
+			description="dolor sit amet, consectetur adipiscing elit. Integer gravida orci lacus, vel convallis enim tposuere ac."
+		/>
+		<InlineMessageDestructive
+			title="Lorem ipsum"
+			withBackground
+			link={{ href: 'https://talend.com', children: 'Learn more' }}
+			description="dolor sit amet, consectetur adipiscing elit. Integer gravida orci lacus, vel convallis enim tposuere ac."
+		/>
+		<InlineMessageBeta
+			title="Lorem ipsum"
+			withBackground
+			link={{ href: 'https://talend.com', children: 'Learn more' }}
+			description="dolor sit amet, consectetur adipiscing elit. Integer gravida orci lacus, vel convallis enim tposuere ac."
+		/>
+	</StackVertical>
+);
 
-export const InformationBackground = {
-	args: {
-		...Information.args,
-		withBackground: true,
-	},
-	render,
+const VariantTemplate: ComponentStory<typeof InlineMessage> = args => {
+	const { variant = 'information', ...rest } = args;
+	return <InlineMessage {...rest} variant={variant} />;
 };
-export const SuccessBackground = {
-	args: {
-		...Success.args,
-		withBackground: true,
-	},
-	render,
-};
-export const WarningBackground = {
-	args: {
-		...Warning.args,
-		withBackground: true,
-	},
-	render,
-};
-export const DestructiveBackground = {
-	args: {
-		...Destructive.args,
-		withBackground: true,
-	},
-	render,
+export const Variant = VariantTemplate.bind({});
+Variant.args = {
+	variant: 'information',
+	title: 'Lorem ipsum',
+	description:
+		'dolor sit amet, consectetur adipiscing elit. Integer gravida orci lacus, vel convallis enim tposuere ac.',
 };
 
-export const InformationBackgroundSmall = {
-	args: {
-		...Information.args,
-		small: true,
-		withBackground: true,
-	},
-	render,
-};
-export const SuccessBackgroundSmall = {
-	args: {
-		...Success.args,
-		small: true,
-		withBackground: true,
-	},
-	render,
-};
-export const WarningBackgroundSmall = {
-	args: {
-		...Warning.args,
-		small: true,
-		withBackground: true,
-	},
-	render,
-};
-export const DestructiveBackgroundSmall = {
-	args: {
-		...Destructive.args,
-		small: true,
-		withBackground: true,
-	},
-	render,
-};
+export const WithRouterLink = () => (
+	<BrowserRouter>
+		<InlineMessage
+			withBackground
+			description="Inline message with a Router Link"
+			variant="information"
```

Comment:
Wondering if this is really useful since you'll most likely want to change all the other props altogether rather than only the type (wording, etc). Also feels **a bit** like an anti-pattern (sort of) to what's mainly done in the DS (i.e exposing variations as components).
It won't prevent me from sleeping though.

Reply:
(erratum, this pattern is also used in Skeleton)

Reply:
Yeah. About this. 
I personally don't like this pattern. I think it's a maintenance trap. Standalone components are easier to make evolve without breaking things. But we asked around and people told us they wanted the collection ones with props. 
We provided.

Reply:
> people told us they wanted

you can use my name

Reply:
Not just you.